### PR TITLE
Increase version to 0.7.0, allow delete programs, make various commands into subcommands

### PR DESCRIPTION
**Pull request by montaguegabe**
I tested all the commands to make sure they worked using the new argparse setup

### CODE REVIEW

**Review by Sand1929 — APPROVED** [review]

**Review by Sand1929 — COMMENTED** [review]
Is this change related to the function definition bug we just found?

**Comment by montaguegabe** [comment]
I still have to change the project.toml to reflect the new entry point I realized

**Comment by montaguegabe** [comment]
> Is this change related to the function definition bug we just found?

I'm not sure which change exactly you are referring to, but I don't think it is.  I'm still not sure exactly how the custom functions were deleted. I may have deleted them through the admin panel as a test and just not be able to remember it.

File: src/parsagon/main.py [hunk 1]
```diff
@@ -85,8 +139,8 @@ def create(task, pipeline_name=None, verbose=False):
                     debug_suffix = f" ({custom_function.name})"
                     description = custom_functions_to_descriptions.get(custom_function.name)
                     description = " to " + description if description else ""
-                    if verbose:
-                        description += debug_suffix
+                    # if verbose:
```
Comment (Sand1929) [inline src/parsagon/main.py:142]: I think the verbose argument got a little messed up in this commit

File: src/parsagon/main.py [hunk 2]
```diff
@@ -19,37 +19,91 @@
...
 
-def create(task, pipeline_name=None, verbose=False):
+    args = parser.parse_args()
+    kwargs = vars(args)
+    func = kwargs.pop("func")
+    verbose = kwargs.pop("verbose")
     configure_logging(verbose)
 
+    if func:
+        func(**kwargs)
```
Comment (Sand1929) [inline src/parsagon/main.py:101]: Also, I just realized: if `run` is only called here, the output of a program is never printed for the user, which makes calling `run` pretty pointless.

Comment (Sand1929) [reply src/parsagon/main.py:101]: which I guess is fixed by saying `return func(**kwargs)` unless I'm missing something

**Pull request merged**